Makefile

```diff
@@ -128,6 +133,10 @@ install-go-bindata:
 	go get github.com/jteeuwen/go-bindata/...
 	go get github.com/elazarl/go-bindata-assetfs/...
 
+.PHONY: install-gas
+install-gas:
+	go get github.com/GoASTScanner/gas/cmd/gas/...
```

Comment:
I am still not clear why we need the extra makefile targets instead of rolling it up into the main `install` and `lint` (extra targets increase noise / entropy)

Reply:
`install` is quite clean, doing only a `glide install`. It does delegate the installation of the dependent tool (`glide`) to a separate target, so, there's a precedent :-)

About `gas` getting called directly from `lint`, I guess it's a matter of taste. When working on fixing the warnings provided by `gas`, I find it easier to run it in isolation as `make gas`.